var/spack/repos/builtin/packages/extrae/package.py

```diff
@@ -35,54 +35,55 @@ class Extrae(Package):
        OpenMP, CUDA, OpenCL, pthread, OmpSs"""
     homepage = "https://tools.bsc.es/extrae"
     url      = "https://ftp.tools.bsc.es/extrae/extrae-3.4.1-src.tar.bz2"
+    version('3.7.1', sha256='95810b057f95e91bfc89813eb8bd320dfe40614fc8e98c63d95c5101c56dd213')
     version('3.4.1', '69001f5cfac46e445d61eeb567bc8844')
 
+    depends_on("autoconf")
```

Comment:
Is `autoconf` actually required? Most of the time it's not if the tarball already comes with a `configure` script.

Reply:
Yes. Configure script provision by upstream is hit and miss in the tarballs.

Reply:
So you're saying some tarballs come with a configure script and some don't? In that case, you'll need more dependencies than just `autoconf`

Reply:
Good point, I have added automake and libtool as well.